vyper/cli/vyper_compile.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python3
+import argparse
+from collections import (
+    OrderedDict,
+)
+import json
+from pathlib import (
+    Path,
+)
+import sys
+from typing import (
+    Any,
+    Iterable,
+    Iterator,
+    Sequence,
+    Set,
+    TypeVar,
+)
+import warnings
+
+import vyper
+from vyper.parser import (
+    parser_utils,
+)
+from vyper.settings import (
+    VYPER_TRACEBACK_LIMIT,
+)
+from vyper.signatures.interface import (
+    extract_file_interface_imports,
+)
+from vyper.typing import (
+    ContractCodes,
+    ContractName,
+)
+
+T = TypeVar('T')
+
+
+def _parse_cli_args():
+
+    warnings.simplefilter('always')
+
+    format_options_help = """Format to print, one or more of:
+    bytecode (default) - Deployable bytecode
+    bytecode_runtime   - Bytecode at runtime
+    abi                - ABI in JSON format
+    abi_python         - ABI in python format
+    ast                - AST in JSON format
+    source_map         - Vyper source map
+    method_identifiers - Dictionary of method signature to method identifier.
+    combined_json      - All of the above format options combined as single JSON output.
+    interface          - Print Vyper interface of a contract
+    external_interface - Print Externa Contract of a contract, to be used as outside contract calls.
```

Comment:
```suggestion
    external_interface - Print External Interface of a contract, to be used as outside contract calls.
```

Typo I noticed (from original)

Reply:
:+1:, i probably should have dedented that too